src/components/WalletConnect/NanoContract/NanoContractMethodArgs.js

```diff
@@ -129,6 +144,60 @@ export const NanoContractMethodArgs = ({ blueprintId, method, ncArgs }) => {
   )
 };
 
+/**
+ * Component resposible to render the appropriate format for the value
+ * taking in consideration the type.
+ *
+ * Remarks
+ * The values received in here when derived from 'byte' type like
+ * 'TxOutputScript', 'TokenUid' and 'VertexId' are already in its
+ * hexadecimal format.
+ *
+ * Values of type 'Address', which also derives from 'byte' are
+ * in its base58 format.
+ *
+ * Values of type 'SignedData[Result]' arrives here using its data
+ * part already.
```

Comment:
What does this mean?

Reply:
I mean, it is already in presentation format.